projects/lux-components-lib/src/lib/lux-action/lux-menu/lux-menu.component.ts

```diff
@@ -193,24 +221,46 @@ export class LuxMenuComponent implements AfterViewInit, AfterContentInit, AfterV
     this.customTriggerEl?.addEventListener('click', this.onTrigger.bind(this));
   }
 
+  ngOnInit() {
+    if (this.luxShowSections() && this.luxDisplayExtended) {
+      console.warn(
+        'Das Menu mit den Abschnitten (LuxShowSections) kann nur angezeigt werden, wenn keine Buttons außerhalb des Menu angezeigt werden können.'
+      );
+    }
+  }
+
   onTrigger() {
     if (this.customTriggerEl && this.triggerButtonDivEl) {
       this.triggerButtonDivEl.click();
     }
   }
 
   ngAfterContentInit() {
+    if (this.luxShowSections()) {
+      this.menuItemsNotExtended = this.menuNotExtendedPanel.toArray().map((ref) => (ref instanceof ElementRef ? ref.nativeElement : ref));
+      this.cdr.detectChanges();
+    }
     this.menuItemChangeSubscription = this.luxMenuItemComponents.changes.subscribe(() => {
-      this.menuItems = this.luxMenuItemComponents.toArray();
-      this.calculateMenuItemWidths();
-      this.updateExtendedMenuItems();
+      if (this.luxShowSections()) {
```

Comment:
Dieser Block kommt zweimal vor und hätte in eine Methode ausgelagert werden können.